docs/_guides/cron-event-driver.md

```diff
@@ -0,0 +1,92 @@
+# Event Driver: cron
+The `cron` event driver fires events on a schedule determined by the cron expression passed to the event driver when created. This allows Dispatch to trigger functions in response to events generated on a scheduled basis.
+
+## Getting Started
+This guide assumes that you have a working Dispatch instance and access to the Dispatch CLI configured to work with your Dispatch instance. If you need helping creating a Dispatch instance or configuring your Dispatch CLI see the quickstart guide [here](../_front/quickstart.md)
```

Comment:
use the jekyl syntax for creating links (a.k.a. `[quickstart]({{ '/documentation/front/quickstart' | relative_url }})`)